src/cli.cpp

```diff
@@ -102,6 +103,8 @@ int main(int argc, char *argv[]) try {
         cxxopts::value<uint32_t>(buffmegabytes))(
         "p, progress", "Display progress percentage during plotting",
         cxxopts::value<bool>(show_progress))(
+        "disable_parallel_read", "Do not use parallel reads - revert back to sequential",
```

Comment:
No short version?

Reply:
I did this on purpose. Do you want this changed to `--parallel_read=false` to turn off?